compiler/ballerina-parser/src/test/resources/misc/annotations/annotations_assert_10.json

```diff
@@ -0,0 +1,132 @@
+{
+  "kind": "MODULE_PART",
+  "hasDiagnostics": true,
+  "children": [
+    {
+      "kind": "LIST",
+      "children": []
+    },
+    {
+      "kind": "LIST",
+      "hasDiagnostics": true,
+      "children": [
+        {
+          "kind": "MODULE_VAR_DECL",
+          "hasDiagnostics": true,
+          "diagnostics": [
+            "ERROR_METADATA_NOT_ATTACHED_TO_A_TOP_LEVEL_CONSTRUCT"
```

Comment:
@nadeeshaan this diagnostic is enough to identify the dummy construct attached to the metadata right?

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/29420?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDE3MjA5ODgyMTQ6MTMyOTY3NA%3D%3D&notifications_query=reason%3Amention#issuecomment-811613853

Reply:
Resolving as per the https://github.com/ballerina-platform/ballerina-lang/issues/29420#issuecomment-811838312